azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -217,7 +217,9 @@ def on_disconnect(client, userdata, rc):
             if not this:
                 # Paho will sometimes call this after we've been garbage collected,  If so, we have to
                 # stop the loop to make sure the Paho thread shuts down.
-                logger.info("disconnected after garbage collection. stopping loop")
+                logger.info(
+                    "on_disconnect called with this=None. Transport must have been garbage collected. stopping loop"
```

Comment:
>this=None [](start = 47, length = 9)

Do you want to say this specifically ? or say `on_disconnect called with transport being None..`